test/linter.js

```diff
@@ -114,7 +114,7 @@ describe('AutoRest Linter validation:', function () {
             prFile += '';
             return prFile.startsWith(configDir) && prFile.indexOf('examples') === -1 && prFile.endsWith('.json');
           }).forEach(prFileInConfigFile => {
-            console.warn(`Configuration file not found for file: ${prFileInConfigFile}, running validation rules against it in individual context.`);
+            console.warn(`WARNING: Configuration file not found for file: ${prFileInConfigFile}, running validation rules against it in individual context.`);
```

Comment:
nit: Wonder if we should integrate this in the validator itself.

Reply:
@dsgouda Maybe, feel free to open an issue on the validator if you think we should move the check there, and we can evaluate. I wouldn't want to change it as part of this change.